DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/streaming_data_interfaces/src/streaming_data_interfaces.cpp

```diff
@@ -0,0 +1,117 @@
+#include <iostream>
+#include <sycl/ext/intel/ac_types/ac_int.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/ext/intel/prototype/pipes_ext.hpp>
+#include <sycl/sycl.hpp>
+
```

Comment:
It's missing the exception handler header, and the it's associated property to be added to the queue